src/click/decorators.py

```diff
@@ -342,19 +342,16 @@ def version_option(
         frame = inspect.currentframe()
         assert frame is not None
         assert frame.f_back is not None
-        f_globals = frame.f_back.f_globals if frame is not None else None
+        assert frame.f_back.f_globals is not None
+        f_globals = frame.f_back.f_globals
         # break reference cycle
         # https://docs.python.org/3/library/inspect.html#the-interpreter-stack
         del frame
```

Comment:
Can collapse this to `f_globals = inspect.currentframe().f_back.f_globals` and remove the `frame = `, `del`, and comment.

Reply:
done